ginga/AstroImage.py

```diff
@@ -643,7 +640,7 @@ def info_xy(self, data_x, data_y, settings):
         ra_lbl, dec_lbl = six.unichr(945), six.unichr(948)
 
         # Calculate WCS coords, if available
-        ts = time.time()
+        #ts = time.time()
```

Comment:
we can probably remove these timing calls.  They were there to measure the slowness of astropy coordinate conversions.  But now that problem is handled elsewhere by just not calling this function as frequently.